packages/fetchai/connections/p2p_libp2p/dht/dhtpeer/dhtpeer.go

```diff
@@ -537,35 +537,54 @@ func (dhtPeer *DHTPeer) lookupAddressDHT(address string) (peer.ID, error) {
 		Msgf("Querying for providers for cid %s...", addressCID.String())
 	ctx, cancel := context.WithTimeout(context.Background(), addressLookupTimeout)
 	defer cancel()
-	providers := dhtPeer.dht.FindProvidersAsync(ctx, addressCID, 1)
+	var elapsed time.Duration
+	var provider peer.AddrInfo
+	var connected bool = false
+	var s network.Stream
```

Comment:
single letter variables are hard to understand for new devs; call it stream?

Reply:
sure